test/e2e/framework/utils/log.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
+)
+
+func nowStamp() string {
+	return time.Now().Format(time.StampMilli)
+}
+
+func log(level string, format string, args ...interface{}) {
+	fmt.Fprintf(ginkgo.GinkgoWriter, nowStamp()+": "+level+": "+format+"\n", args...)
+}
+
+func Logf(format string, args ...interface{}) {
+	log("INFO", format, args...)
+}
+
+func Failf(format string, args ...interface{}) {
+	msg := fmt.Sprintf(format, args...)
+	ginkgo.Fail(msg)
+}
```

Comment:
Possible to just use something like logrus to eliminate needing this?

Reply:
humm, do we need to do structured logging in test cases? this can be an improvement :D

Reply:
Structured as in JSON: no probably not, but the default format is highly similar to what you've defined here.

Reply:
switched to use zap xD